arctic/store/_ndarray_store.py

```diff
@@ -163,6 +158,41 @@ def get_info(self, version):
         ret['rows'] = int(version['up_to'])
         return ret
 
+    def _chunked_read(self, arctic_lib, version, symbol, date_range, read_preference=None):
```

Comment:
So looks like this shares the unpacking logic with `_do_read` the difference being the query spec